crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -1111,7 +1122,10 @@ impl From<Option<enums::CaptureMethod>> for TransactionType {
     fn from(capture_method: Option<enums::CaptureMethod>) -> Self {
         match capture_method {
             Some(enums::CaptureMethod::Manual) => Self::Authorization,
-            _ => Self::Payment,
+            Some(enums::CaptureMethod::Automatic)
+            | Some(enums::CaptureMethod::ManualMultiple)
+            | Some(enums::CaptureMethod::Scheduled)
```

Comment:
We should throw 4xx in case of ManualMultiple & Scheduled

Reply:
Done